openslides/users/static/js/users/site.js

```diff
@@ -458,6 +475,62 @@ angular.module('OpenSlidesApp.users.site', [
             }
             $scope.sortColumn = column;
         };
+        // define custom search filter string
+        $scope.getFilterString = function (user) {
```

Comment:
Yes but you can handle a list of property names that you want to use in the filter and a generic function cares about creating the whitespace join. So we do not have to program the logic of joining strings with whitespace twice.